### PR TITLE
223 add uss delete

### PR DESCRIPTION
**Pull request by Colin-Stone**

### CODE REVIEW

**Comment by Colin-Stone** [comment]
Based upon the equivalent function for Datasets. Implemented API and CLI and tests for unit, integration(CLI only) and system.

File: packages/zosfiles/__tests__/__system__/cli/delete/uss/uss.files.delete.directory.system.test.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            testName: "zos_delete_uss_directory"
+        });
+
+        systemProps = new TestProperties(TEST_ENVIRONMENT.systemTestProperties);
+        defaultSystem = systemProps.getDefaultSystem();
+
+        REAL_SESSION = TestEnvironment.createZosmfSession(TEST_ENVIRONMENT);
+        user = defaultSystem.zosmf.user.trim();
+        ussname = `${defaultSystem.zosmf.user.trim()}.aTestUssDirectory`;
+        ussname = ussname.replace(/\./g, "");
```
Comment (ChrisBoehmCA) [inline packages/zosfiles/__tests__/__system__/cli/delete/uss/uss.files.delete.directory.system.test.ts]: I didn't exactly follow why you replace `.` after adding one in the line above. Probably not a problem but wanted to understand the rationale

File: packages/zosfiles/__tests__/__system__/cli/delete/uss/uss.files.delete.directory.system.test.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
+/*
```
Comment (ChrisBoehmCA) [inline packages/zosfiles/__tests__/__system__/cli/delete/uss/uss.files.delete.directory.system.test.ts]: These two test files deviate slightly from the pattern we used for the other commands. Other test file names begin with `cli.` and the rest of the filename mimics the command structure e.g. `cli.files.delete.uss.directory.system.test.ts`. Something similar to that would be good. Good job on the tests though, they all passed for me and look good.

Comment (Colin-Stone) [reply packages/zosfiles/__tests__/__system__/cli/delete/uss/uss.files.delete.directory.system.test.ts]: Ah yes. That's makes sense.

File: packages/zosfiles/__tests__/__system__/cli/delete/uss/uss.files.delete.directory.system.test.ts (hunk 1 above)
Comment (Colin-Stone) [reply packages/zosfiles/__tests__/__system__/cli/delete/uss/uss.files.delete.directory.system.test.ts]: Hmm.. Neither do I. I reckon I've copied it from the dataset delete along with the overall flow and didn't ask myself why. My bad.

**Pull request merged**